container/test-impl-base/src/main/java/org/jboss/arquillian/container/test/impl/RunModeUtils.java

```diff
@@ -38,7 +40,7 @@
    private RunModeUtils() { }
    
    /**
-    * Check is this should run as client.
+    * Returns if the given test should run as client.
```

Comment:
I'm not sure if this statement is really readable. How about `Verifies if the given test should be ran in the client mode`?